From 7a31eca88aed8cb486f892185d363fe6e8b8773c Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Wed, 4 Dec 2024 15:00:14 +0800
Subject: [PATCH 191/296] AOSCOS: MIPS: ec_wpce775l: add a missing prototype
 for ec_query_get_event_num()

Add missing prototype for ec_query_get_event_num() to suppress a -Werror
for "no previous prototype".

Fixes: "AOSCOS: MIPS: Loongson 3: Add basic EC operations"
Signed-off-by: Mingcong Bai <jeffbai@aosc.io>

Signed-off-by: Kexy Biscuit <kexybiscuit@aosc.io>
---
 arch/mips/include/asm/mach-loongson64/ec_wpce775l.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/mips/include/asm/mach-loongson64/ec_wpce775l.h b/arch/mips/include/asm/mach-loongson64/ec_wpce775l.h
index 3d2ca3245a3e..fc60026bc674 100644
--- a/arch/mips/include/asm/mach-loongson64/ec_wpce775l.h
+++ b/arch/mips/include/asm/mach-loongson64/ec_wpce775l.h
@@ -374,6 +374,7 @@ extern int ec_write_noindex(unsigned char command, unsigned char data);
 
 /* Query sequence of 62/66 port access routine. */
 extern int ec_query_seq(unsigned char command);
+extern int ec_query_get_event_num(void);
 extern int ec_get_event_num(void);
 
 extern void clean_ec_event_status(void);
-- 
2.48.1

